chart/compass/charts/gateway/values.yaml

```diff
@@ -38,3 +38,7 @@ rateLimit:
 
 metrics:
   port: 3001
+
+auditLog:
+  maxBodySizeBytes: 998000
```

Comment:
Why exactly 998000?

Reply:
because the current auditlog limit is 1MB and I want to be a bit less than that, becasue the auditlog message contains additional information, not just the body